src/main/java/utilities/ResponseHandler.java

```diff
@@ -0,0 +1,187 @@
+        HashMap<String, String> responseMap = new HashMap<>();
+        responseMap.put("byee", "You meant bye?");
+        responseMap.put("byeee", "You meant bye?");
+        responseMap.put("hi", "Hi again!");
+        return responseMap.getOrDefault(message, message);
+    }
+
+    /**
+     * Prints a line separator to the console.
+     *
+     * @return A line separator.
+     */
+    public static String printLine() {
+        return "____________________________________________________________";
+    }
+
+    /**
+     * Generates a greeting message.
+     *
+     * @param chatBotName The name of the chatbot.
+     * @return A greeting message.
+     */
+    public static String greeting(String chatBotName) {
+        return "Hello! I'm " + chatBotName + "\nWhat can I do for you?";
+    }
+
+    /**
+     * Generates a goodbye message.
+     *
+     * @return A goodbye message.
+     */
+    public static String bye() {
+        return "Goodbye. Hope to see you again!";
+    }
+
+    /**
+     * Generates a command message.
+     *
+     * @param command The command to be printed.
+     * @return The command message.
+     */
+    public static String commandPrint(String command) {
+        return command;
+    }
+
+    /**
+     * Generates a message with information about a task and the total number of tasks.
+     *
+     * @param task The task to be printed.
+     * @param size The total number of tasks.
+     * @return A message with task information.
+     */
+    public static String commandPrint(Task task, int size) {
+        return "Got it. I've added this task:\n" + task
+                + "\nNow you have " + size
+                + " tasks in the list.";
+    }
+
+    /**
+     * Generates a message with a list of tasks.
+     *
+     * @param list The list of tasks to be printed.
+     * @return A message with the list of tasks.
+     */
+    public static String commandListPrint(ArrayList<Task> list) {
+        if (list.isEmpty()) {
+            return "Task list is empty :(";
+        }
+        StringBuilder stringOfAddedCommands = new StringBuilder();
+        for (int i = 0; i < list.size(); i += 1) {
+            stringOfAddedCommands.append(i + 1 + ". ");
+            stringOfAddedCommands.append("\n");
+            stringOfAddedCommands.append(list.get(i));
+            if (i != list.size() - 1) {
+                stringOfAddedCommands.append("\n");
+            }
+        }
+        return stringOfAddedCommands.toString();
+    }
+
+    /**
+     * Generates a message indicating the result of a mark action on a task.
+     *
+     * @param action The action performed ("mark" or "unmark").
+     * @param task   The task on which the action was performed.
+     * @return A message with the mark action result.
+     */
+    public static String markActionPrint(String action, Task task) {
+        StringBuilder stringResponse = new StringBuilder();
+        if (action.equals("mark")) {
```

Comment:
perhaps you could add a toLowerCase() method to account for cases where user inputs Mark instead of mark